modules/UI.js

```diff
@@ -0,0 +1,109 @@
+import Task from './task.js';
+
+const { getStorage, setStorage } = require('./storage.js');
+
+export const todoContainer = document.querySelector('.todo-container');
+const userInput = document.querySelector('#input');
+
+export default class UI {
+  static createTodo = (todo) => {
+    const todoList = document.createElement('li');
+    todoList.setAttribute('id', todo.index);
+    todoList.setAttribute('class', 'list-item');
+    todoList.innerHTML = `<div class ="flex"><input type="checkbox" class ="check">
+    <input type ="text" class="text" value = "${todo.description}"/></div> <span class = "material-icons gray vertical">more_vertical</span>`;
+    todoContainer.appendChild(todoList);
+  }
+
+  static showBook = (event) => {
+    event.preventDefault();
+    const array = getStorage();
+    const description = userInput.value;
+    const index = (array.length + 1).toString();
+    const completed = false;
+    const todo = new Task(description, completed, index);
+    array.push(todo);
+    setStorage(array);
+    UI.createTodo(todo);
+    UI.clearField();
+  }
+
+  static renderTodo = () => {
+    const todos = getStorage();
+    todos.forEach((todo) => {
+      UI.createTodo(todo);
+    });
+  }
+
+  static editText = (newInput, oldValue) => {
+    const localStore = getStorage();
+    localStore.forEach((task) => {
+      if (oldValue === task.description) {
+        task.description = newInput;
+      }
+    });
+    setStorage(localStore);
+  }
+
+  static updateIndex = (index, array) => {
+    for (let i = 0; i < array.length; i += 1) {
+      if (index < array[i].index) {
+        array[i].index -= 1;
+      }
+    }
+    return array;
+  }
+
+  static deleteTodo = (event) => {
+    const todos = getStorage();
+    let found = null;
+    const li = event.target.parentElement.children[0];
+    todos.forEach((todo) => {
+      if (li.children[1].value === todo.description) {
+        event.target.parentElement.remove();
+        found = todo;
+      }
+    });
+    if (found != null) {
+      const index = todos.indexOf(found);
+      todos.splice(index, 1);
+      UI.updateIndex(found.index, todos);
+    }
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
```

Comment:
- [ ]  [Optional] It will be nice if the delete button is visible on the page without editing or when the three dots is hovered. presently, one can only see the delete button when editing.